recipes/_ec2_udev_rules.rb

```diff
@@ -20,6 +20,13 @@
   mode '0644'
 end
 
+cookbook_file 'ebsnvme-id' do
+  path '/sbin/ebsnvme-id'
```

Comment:
Let's install this in /usr/local/sbin, and maybe call it cfncluster-ebsnvme-id just in case the other gets installed by some other package and conflicts.

Reply:
Ya addressed both, updated path and script name.